src/Schema/Keywords/Type.php

```diff
@@ -46,10 +65,29 @@ public function validate($data, string $type, ?string $format = null): void
 
                 break;
             case CebeType::ARRAY:
-                if (! is_array($data) || ArrayHelper::isAssoc($data)) {
+                $additionalProperties = $this->parentSchema->items
```

Comment:
I'm not sure whether I got this expression. `items` could be a scalar type which cannot have additional properties